doc/spec/source/views/iota_view.rst

```diff
@@ -0,0 +1,24 @@
+.. SPDX-FileCopyrightText: Intel Corporation
+..
+.. SPDX-License-Identifier: BSD-3-Clause
+
+.. include:: ../include/distributed-ranges.rst
+
+.. _iota_view:
+
+===========
+ iota_view
+===========
+
+Interface
+=========
+
+.. doxygenvariable:: dr::mhp::views::iota
```

Comment:
We document the interface, not the implementation. So we should document `dr::views::iota` and refer to that. It would be good if the `dr::views` namespace is not shown